syntax = "proto3";

package queue;

message Request {
    string video_id = 1;
}

message QueueStatus {
    string status = 1;
    string listener = 2;
}

service Queue {
    rpc check_queue(Request) returns (QueueStatus) {};
}
